packages/amplify-category-api/src/provider-utils/awscloudformation/cfn-api-artifact-handler.ts

```diff
@@ -260,6 +260,7 @@ class CfnApiArtifactHandler implements ApiArtifactHandler {
  */
 export const writeResolverConfig = async (conflictResolution: ConflictResolution, resourceDir) => {
   const localTransformerConfig = await readTransformerConfiguration(resourceDir);
-  localTransformerConfig.ResolverConfig = conflictResolutionToResolverConfig(conflictResolution);
+  const resolverConfig = conflictResolutionToResolverConfig(conflictResolution);
+  if (!_.isEmpty(resolverConfig)) localTransformerConfig.ResolverConfig = resolverConfig;
```

Comment:
does this work if you don't have a resolver config and you try to enable datastore?

Reply:
I tried adding DataStore and then removing it, it doesn't look like it removes Resolver Config when DataStore is removed but do we want to delete tables when DataStore is removed?